src/vg.proto

```diff
@@ -245,6 +257,7 @@ message SnarlTraversal {
 
     // A Snarl with the start and end Visits filled in to label which Snarl these traverse.
     Snarl snarl = 2;
+    string name = 3;
```

Comment:
The name is for the SnarlTraversal - each alt_path has a direct mapping to a SnarlTraversal. The Snarl names correspond to variant names.